package.json

```diff
@@ -4,7 +4,14 @@
   "description": "",
   "main": "index.js",
   "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
+    "client": "yarn workspace @course-tracker/client start",
+    "build": "yarn workspace @course-tracker/client build",
+    "dev": "yarn workspace @course-tracker/server dev",
+    "start": "concurrently --kill-others-on-fail \"yarn dev\" \"yarn client\" \"yarn build\"",
```

Comment:
Hmm, ideally `start` would be the production command. Which'll only need to run the server in production.